examples/bigcommerce/lib/bigcommerce/bigcommerce.ts

```diff
@@ -0,0 +1,195 @@
+const STORE_NAME = process.env.BIGCOMMERCE_STORE_NAME
+const ACCESS_TOKEN = process.env.BIGCOMMERCE_ACCESS_TOKEN
+
+const getBearerToken = async () => {
+  const A_WEEK_FROM_NOW = Math.floor(
+    new Date(new Date().getTime() + 1000 * 60 * 60 * 24 * 7).getTime() / 1000,
+  )
+
+  const response = await fetch(
+    `https://api.bigcommerce.com/stores/${process.env.BIGCOMMERCE_STORE_HASH}/v3/storefront/api-token`,
+    {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Auth-Token': ACCESS_TOKEN ?? '',
```

Comment:
Should this function fail early if the `BIGCOMMERCE_ACCESS_TOKEN` env var isn't defined?